src/workerd/io/cdp.capnp

```diff
@@ -391,9 +383,7 @@ struct Command $Json.discriminator(name = "method") {
     profilerEnable @6 :Method(Profiler.Command.Enable.Params, Profiler.Command.Enable.Result) $Json.name("Profiler.enable") $Json.flatten();
     profilerStart @7 :Method(Profiler.Command.Start.Params, Profiler.Command.Start.Result) $Json.name("Profiler.start") $Json.flatten();
     profilerStop @8 :Method(Profiler.Command.Stop.Params, Profiler.Command.Stop.Result) $Json.name("Profiler.stop") $Json.flatten();
-    heapProfilerEnable @9 : Method(HeapProfiler.Command.Enable.Params, HeapProfiler.Command.Enable.Result) $Json.name("HeapProfiler.enable") $Json.flatten();
```

Comment:
Why remove these?

Reply:
Oh, sorry, meant to add a comment in the PR. These aren't needed at all. Better to just pass them through. If we decide later to try to support the heap profiler allocation timeline in edgeworker in the untrusted mode we can add these back but for now it's just not something we need.

Reply:
It looks like previously we were handling these commands by... reflecting them back to the client, without passing them through to V8? But now we will be actually handing them to V8?

I don't understand the old behavior... what was it trying to do?

Reply:
It was incomplete and really wasn't doing anything. I had been trying to get the sampling mechanism working in the untrusted mode but hadn't got back around to it because it hadn't been a priority.